lib/views/help/privacy.html.erb

```diff
@@ -560,7 +560,7 @@
   <p>
     In most cases our legal basis for processing personal information is
     "legitimate interest" (this is as laid out in <a
-    href="https://gdpr-info.eu/art-6-gdpr/">6(1)(f)</a> – of the GDPR, in force
+    href="https://www.legislation.gov.uk/eur/2016/679/article/6">6(1)(f)</a> – of the UK GDPR, in force
     from 25 May 2018). We believe that we are pursuing a legitimate interest in
```

Comment:
Is this still the correct date to use now that we've transitioned to _UK_ GDPR?

Reply:
I think it'd be best if we dropped the date - GDPR has been in effect in its guises since 2018, so it probably isn't necessary to continue referring to it as if we are noting it as being new / different.

If you're happy with that suggestion @garethrees, I'll push a patch?

Reply:
Yeah, don't think the date is adding anything at this point. Thanks so much!

Reply:
> Yeah, don't think the date is adding anything at this point. Thanks so much!

Pushed as 0540db1 - this should now be good to go 🤞